editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -550,6 +558,7 @@ void TileMapEditor::_draw_fill_preview(int p_cell, const Point2i &p_point, bool
 	PoolVector<Vector2> points = _bucket_fill(p_point, false, true);
 	PoolVector<Vector2>::Read pr = points.read();
 	int len = points.size();
+	int time_after = OS::get_singleton()->get_ticks_msec();
```

Comment:
Leftover from debugging? Doesn't seem used anywhere.

Reply:
Wat!? I can't remember putting that there as I did not time anything. How did this end up in my code?

I'll remove that one and try updating the pull request. Thank you for pointing it out!